tests/integrations/django/test_basic.py

```diff
@@ -1029,6 +1027,8 @@ def test_cache_spans_middleware(
         ],
         traces_sample_rate=1.0,
     )
+
+    client.application.load_middleware()
```

Comment:
By changing the default it was uncovered, that this test only worked by coincidence. Had to load the middleware after sdk init, to have the options set in DjangoIntegration.